Add round 2 chaser email (#23)
srobo/team-emails: SR2019/2019-07-02-kit-chase-round-2.md

---
to: Teams with still outstanding kit

> subject: Outstanding kit - Round 2

Reviewer: "Round 2" reads a bit oddly, perhaps use something more prose-like:
```suggestion
subject: Outstanding kit from SR2019 (second followup)
```

---

Hello!

We sent an email to you back in May stating we required a kit back from you. Unfortunately, we've not heard a response.

> The kits are loaned to you fur the duration of the competition year, and as this has ended, we need the kits back, to prepare for future events.

Reviewer: I wonder whether we should include something explicitly saying how them keeping a kit may mean that it prevents others from taking part in the future?

> The kits are loaned to you fur the duration of the competition year, and as this has ended, we need the kits back, to prepare for future events.

Author: That feels a little too guilt trip-y for my liking

> Please securely pack the kit and ship it to the address listed on [our website](https://studentrobotics.org/contact/). Once you have dispatched it, please [let us know](mailto:teams@studentrobotics.org).

Reviewer 2: Nowadays most people don't have working mailto links, maybe better to explicitly say "let us know by emailing us at <email>"?

Author: In my experience, most people either read the email from the link, or just respond to the email, which will come from that address anyway

## How to ship your kit

> - Kits should be shipped in the white 18l Really Useful Box (RUB). For extra protection, we recommend wrapping this in paper / cling film during shipping.

Reviewer 2: Mention the box is what the kit originally came in

- All parts of the kit should be suitably protected either in Jiffy bags or bubble wrap.
- Damaged batteries must not be shipped. If you have concerns about your batteries, please contact us.
- Any empty space in the box should be filled with paper or bubble wrap. Please don't use packing peanuts as they're messy.

> Please insure the kit during shipping and use a courier which supports tracking, to ensure the kit isn't lost.

Reviewer 2: ```suggestion
Please ensure the kit during shipping and use a courier which supports tracking, to ensure the kit isn't lost.
```

Reviewer 2: Maybe also mention if the kit is lost they're expected to repay us? This is the mean email anyway.

Reviewer: Regarding the first comment in this chain - I'm pretty sure the original text is correct.:
- "insure" is the verb form of obtaining insurance, which is the intent here
- "ensure" means "make happen"  (note the later usage of this in the same sentence, which is also correct)

Author: "Insure" is correct here.

I think i'm ok with leaving out the fact they have to repay us for it. This isn't actually the mean email, the properly mean one is when we start contacting people higher up in the school.

> Kit data for couriers:

Reviewer 2: ```suggestion
Kit information for couriers:
```

- Dimensions: 48cm x 39cm x 20cm
- Weight: 6kg
- Value (for insurance): £500
- Batteries: Lithium Ion. Packed with equipment. Batteries less than 100Wh.

> If we do not hear back within 2 weeks, we will be forced to take this higher. If you have any questions, please let us know as soon as possible!

Reviewer 2: ```suggestion
If we do not hear back within 2 weeks, we will be forced to contact the head of your school/organisation directly. If you have any questions, please let us know as soon as possible!
```
